hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaTenantConfigurationManagement.java

```diff
@@ -188,6 +192,7 @@
     private void assertValueChangeIsAllowed(final String key, final JpaTenantConfiguration valueChange) {
```

Comment:
Is the key String here really needed? Is it not always equal to the key of the JpaTenantConfiguration object? This would make the assertion methods simpler, including the newly added one